### DISCUSSION TITLE
How to add records in a relation field

### DISCUSSION THREAD

**Discussion by fcrios145** [pocketbase/pocketbase#4508] [Q&A] [answered]
Here's the thing
Let's say that I have two collections
- Quotes
- QuotesDetails

QuotesDetails has a relation **single field** to their master QuoteTable

How can I create a Quotes record with the details in it.
Without calling the QuotesDetails create, multiple times

**Comment by ganigeorgiev** [comment, marked as answer]
For more details, please refer to https://pocketbase.io/docs/working-with-relations/.


~You'll have to first create the "QuotesDetails" records and assign the created ids to their "Quotes" record `relation` field.~

On second read it is not clear what is your collections structure (you have 3 related quotes tables?).

But in any case if you are asking for bulk/transactional web APIs this is not supported at the moment (at least not out of the box).
